cms/static/sass/elements/_modal-window.scss

```diff
@@ -183,6 +217,29 @@
           @extend %t-icon4;
         }
       }
+
+      .subsection-share-link-container {
+        display: flex;
+        align-items: center;
+
+        img {
```

Comment:
I would use a class for the image, if you add another image for some reason the other one will have the same style

Reply:
Done